shopify/resources/access_scope.py

```diff
@@ -2,4 +2,6 @@
 
 
 class AccessScope(ShopifyResource):
-    _prefix_source = "/admin/oauth/"
+    @classmethod
+    def override_prefix(cls):
+        return "/admin/oauth"
```

Comment:
I was about to say "/admin/oauth is not versioned", but it looks like it's currently in the versioned routes. I don't think it should be, since gaining access to the Admin API is different than the (versioned) resources it exposes.

I think we should remove these scopes from `versioned_routes`. Make sense @eapache @joeyzhoucode ?

https://github.com/Shopify/shopify/blob/master/config/routes/admin.rb#L177-L192

Reply:
So everything under here: https://github.com/Shopify/shopify/blob/15d5af2145d6931f875d1f72c72f4e47bb0bed28/config/routes/admin.rb#L177

Reply:
We're going to leave it un-versioned.